### ISSUE TITLE
`formulae` on conda-forge has version 0.0.0

### ISSUE THREAD

**Issue by digicosmos86** [bambinos/formulae#111]
Hi @tomicapretto!

We are trying to get `HSSM` onto conda-forge, for which `bambi` is a dependency. However, we get the following error message when `pip check` is run after the package is built in CI: `bambi 0.13.0 has requirement formulae>=0.5.0, but you have formulae 0.0.0.`

After looking into this issue, we found that after installing `formulae` into a fresh conda environment and doing `pip list`, the version for `formulae` is shown as 0.0.0, which could be the culprit of this issue:

```
conda install -c conda-forge formulae
pip list
```

```
Package         Version
--------------- -------
formulae        0.0.0
numpy           1.26.4
pandas          2.2.2
pip             24.0
python-dateutil 2.9.0
pytz            2024.1
scipy           1.13.0
setuptools      69.5.1
six             1.16.0
tzdata          2024.1
wheel           0.43.0
```

Could you look into this? Thank you!

**Comment by digicosmos86** [comment]
Hi @tomicapretto!

I did some further investigation and found this to be a very puzzling issue. I downloaded the sdist of formulae 0.5.3 from PyPI. I installed it with pip, and then also tried to build the wheel from the source extracted from the sdist. The version was correctly recognized. However, it was clear from the logs of the build process on the feedstock that the version was not correctly recognized. I'll ask the conda-forge folks and see what recommendation that they have. Then I'll do a PR on the feedstock.
